### ISSUE TITLE
[Bug]:  update_subscribed key error is_live

### ISSUE THREAD

**Issue by Zyzto** [tubearchivist/tubearchivist#336]
### Latest and Greatest

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).

### Operating System

Unraid

## Describe the bug
trying to to fetch new videos from subscribed channles will result in error in logs
## Steps To Reproduce
- have correct cookies configered
- go to downloads and Rescan subscriptions
## Expected behavior
to continue without issue and have more detailed error description

### Relevant log output

```shell
[2022-10-15 10:02:08,774: ERROR/ForkPoolWorker-16] Task update_subscribed[b7e745d3-f967-4cea-a884-3b40f3fcda6a] raised unexpected: KeyError('is_live')

Traceback (most recent call last):
File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
R = retval = fun(*args, **kwargs)
File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
return self.run(*args, **kwargs)
File "/app/home/tasks.py", line 68, in update_subscribed
pending_handler.add_to_pending()
File "/app/home/src/download/queue.py", line 200, in add_to_pending
video_details = self.get_youtube_details(youtube_id)
File "/app/home/src/download/queue.py", line 249, in get_youtube_details
if vid["is_live"]:
KeyError: 'is_live'
```

**Comment by bbilly1** [comment]
Thanks for raising this. I have encountered the same error, fix is already incoming: https://github.com/tubearchivist/tubearchivist/commit/a8c5773f816ea3cdf53b4f530a50650cd3abb556

You can make the changes manually yourself, or wait for the next release.

**Comment by bbilly1** [comment]
Thanks for reporting, fix is in v0.2.3.
